src/main/java/org/jpeek/App.java

```diff
@@ -348,6 +360,9 @@ public void analyze() throws IOException {
             App.xsl("matrix.xsl").transform(matrix).toString(),
             "matrix.html"
         );
+        if (metrics.size() == 8) {
```

Comment:
@iMaks99 this condition looks strangely hard-coded number. What if we modify the code above later and make it 9, the report will stop printing?

Reply:
I think we can change condition to better one, which will be depend directly on metric. Now our methodology is based on 8 metrics listed in reports, so in case of modifying metric evaluation or adding new one the following should be done:

1. Modifying total value evaluation according to metric's features.
2. Adding metric to list of metrics. For example for CCM metric: 
```
if (this.params.containsKey("CCM")) {
    metrics.add("CCM"); <---
    reports.add(
        new XslReport(
            chain.transform(skeleton), xsl,
            new ReportData("CCM")
        )
    );
}
```
3. Add if its necessary another check to updated condition:
```
if (this.params.containsKey("LCOM2") && this.params.containsKey("LCOM5")
    && this.params.containsKey("CCM") && this.params.containsKey("NHD")
    && this.params.containsKey("MMAC") && this.params.containsKey("LCC")
    && this.params.containsKey("TCC") && this.params.containsKey("PCC")) {
    new GeneralCalculation().createReport(this.output, metrics);
}
```